### PR TITLE
wip: sequelize boilerplate for votes

### PR DESCRIPTION
**Pull request by kimerran**
let's discuss these sample codes. some directories made by Sequelize.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Please make sure that every `.js` files are in strict mode.

**Review by ghost — APPROVED** [review]

File: controllers/vote_controller.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    models.Vote
+      .build({
+        direction: 'up',
+        vote_to: threadId,
+      })
+      .save()
+      .then(() => {
+        res.send({ result: 'ok' });
+      });
```
Comment (ac-opensource) [inline controllers/vote_controller.js]: No error handling?

**Review by ac-opensource — CHANGES_REQUESTED** [review]

**Review by TheoryOfNekomata — COMMENTED** [review]
Consider using ES6 features for `models/index.js`.

With that said, ESLint with ecmaVersion: 2017 should be used.

**Comment by kimerran** [comment]
@Temoto-kun can you specify which lines needed to be in ES6. And the reason why the need for ES2017? thanks

**Comment by TheoryOfNekomata** [comment]
The whole file is in ES5. Compare with other files in the folder. Anyway, it's not much of a showstopper.

**Review by ac-opensource — APPROVED** [review]

**Pull request merged**